sanic_jwt/initialization.py

```diff
@@ -184,6 +184,10 @@ def __add_endpoints(self):
             url_prefix = self._get_url_prefix()
             self.instance.blueprint(self.bp, url_prefix=url_prefix)
 
+            self.instance.exception(exceptions.Unauthorized)(
+                self.responses.exception_response
+            )
```

Comment:
I like this better than my solution of changing line 179 from `self.bp` to `self.app`. Maybe I should change that back.